test/jdk/java/lang/Character/Latin1CaseFolding.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.fail;
+
+/**
+ * @test
+ * @summary Provides exchaustive verification of Character.toUpperCase and Character.toLowerCase
```

Comment:
typo: "exhaustive"?

Reply:
I did an 'exchaustive' search for 'exchaustive' across the code base and found a two comments in `LocaleData` and `LocaleData.cldr` in jdk/test/jdk/sun/text/resources`.

Would you like me to update these as well while we're here, or should we avoid getting out the scope for this PR?

Reply:
I'd appreciate it. I don't mind fixing it with this PR.

Reply:
Thanks Naoto, I have fixed the spelling in these two unrelated files.